lib/server.js

```diff
@@ -271,6 +271,20 @@ function create(log, error, config, routes, db) {
     }
   )
 
+  const metricsContext = require('./metrics/context')(log, config)
+
+  server.decorate('request', 'stashMetricsContext', request => {
```

Comment:
> I suppose comments at the `metricsContext` methods could say `// this === request` or something...

I just realised jsdoc has [`@this`](http://usejsdoc.org/tags-this.html) for this kind of thing, I'll use that. Thanks!